source/io/JavaScriptInvoke.cpp

```diff
@@ -46,7 +46,7 @@ StringRef AllocateReturnBuffer()
 
 //------------------------------------------------------------
 //! Return parameter type name for the given element(index) in the parameters array
-VIREO_EXPORT const char* JavaScriptInvoke_GetParameterType(StaticTypeAndData *parameters, Int32 index)
+VIREO_EXPORT const char* JavaScriptInvoke_GetParameterTypeName(StaticTypeAndData *parameters, Int32 index)
```

Comment:
Consider deleting JavaScriptInvoke_GetParameterTypeName and JavaScriptInvoke_GetParameterPointer. They appear not to be used anymore.

Reply:
Good catch. Will do.